### ISSUE TITLE
Another-video-player dialog rendered outside of the TV viewport

### ISSUE THREAD

**Issue by dhlavaty** [nova-video-player/aos-AVP#408]
TL;DR; Use-another-video-player dialog is outside of the TV's viewport, so I cannot see all video players available to me.

Long version:

After I have turned on "**Allow-using-another-video-player**" feature in Settings. There is a choose-another-video-player dialog before every video. It is positioned on the bottom of the TV, but outside of the viewport. I'm able to see only first two players available (*But I have at least 5 in my system: "Nova Video Player", "ComMediaPlayer_Video", "VLC", "MX Player", "Kodi"*). Ordering of players in this dialog is also changing, so it is very hard to choose the one I really want.

![TCL-Nova-Video-Player](https://user-images.githubusercontent.com/513138/103537861-6fef9d00-4e95-11eb-92da-7bc89010caca.GIF)

My setup:

AndroidTV (version 9) on TCL 55C815
Nova Video Player 5.15.13-20201203.2027

**Comment by courville** [comment]
I am afraid that this dialog is managed by the platform itself and not by nova.
Thus no easy solution there.
